packages/story-editor/src/components/panels/design/animation/animation.js

```diff
@@ -251,16 +256,21 @@ function AnimationPanel({
             direction={getEffectDirection(updatedAnimations[0])}
             selectedEffectType={updatedAnimations[0]?.type}
             selectButtonStylesOverride={highlight?.focus && styles.OUTLINE}
+            disabled={isFirstPage}
           />
         </StyledRow>
         {updatedAnimations[0] && (
           <EffectPanel
             animation={updatedAnimations[0]}
             onChange={handlePanelChange}
             disabledTypeOptionsMap={disabledTypeOptionsMap}
+            disabled={isFirstPage}
           />
         )}
       </GroupWrapper>
+      {isFirstPage && (
+        <Warning message="Animations will not show up on the first page of a Story because of AMP restrictions." />
```

Comment:
⚠️  All user-facing messages need to be translatable.

Use `__()` from the `i18n` package to do so.

Also, note that the wording on this was not final, as mentioned in #11321.

Let's use something like this:

```
Animations on the first page of a story are not supported for user experience and performance reasons.
```